apps/portal/src/App.js

```diff
@@ -115,6 +124,27 @@ export default class App extends React.Component {
         }
     }
 
+    // User for adding trailing margin to prevent layout shift when popup appears
+    getScrollbarWidth() {
+        // Create a temporary div
+        const outer = document.createElement('div');
+        outer.style.visibility = 'hidden';
+        outer.style.overflow = 'scroll'; // forcing scrollbar to appear
+        document.body.appendChild(outer);
+
+        // Create an inner div
+        const inner = document.createElement('div');
+        outer.appendChild(inner);
+
+        // Calculate the width difference
+        const scrollbarWidth = outer.offsetWidth - inner.offsetWidth;
+
+        // Clean up
+        outer.parentNode.removeChild(outer);
```

Comment:
Small improvement idea: use `clientWidth` instead of creating an inner element:

```
    ...
    document.body.appendChild(div);
    
    const scrollbarWidth = div.offsetWidth - div.clientWidth;
    
    document.body.removeChild(div);
    ...
 ```

Reply:
Great idea! Added the optimization [here](https://github.com/TryGhost/Ghost/pull/21895/commits/fb2bdac616ea34ed7fbee69278068c29750eb03a#diff-e1eb9ae4190f8d5a9a3d7401f75a79cc08e8d12e7e175c9104b91a648aa845f9R132).